.github/workflows/translations.yaml

```diff
@@ -0,0 +1,56 @@
+name: Translations
+
+on:
+  push:
+    branches:
+      - main
+      - 'releases/**'
+  pull_request:
+    branches:
+      - main
+      - 'releases/**'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  translations:
+    name: Translations
+    runs-on: ubuntu-20.04
+    env:
+      QTVERSION: 6.2.4
+
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+
+      - name: Checkout submodules
+        shell: bash
```

Comment:
That should be already the default on non Windows.

Reply:
To be clear, I mean that this line is not necessary as far as I understand, since we're only running Linux ;-)